# Pathogenic_Classifier
##Guide
###With the widespread application of next-generation sequencing technologies in medical genetics, a large amount of DNA variant data has been obtained. For these technologies to have a greater medical impact, we must be able to reliably identify true pathogenic variants from the broader background of benign genetic variation present in the human genome. Many prioritisation tools exist to help with predicting the disease-implicated of missense variants (i.e. sequence alterations that lead to a single amino acid change). Each of these tools will assign scores to individual DNA changes in an attempt to predict whether it is benign or functional. Examples of such tools include REVEL and VEST4. The purpose of the project is to use gene-level information to improve the pathogenicity prediction of these two recognized variant-level classifiers. Genes associated with cataract and retinal disorders genes were studied. DNA variant data were obtained using two databases,  HGMD and gnomAD. This analysis focused on REVEL and VEST4 scores and we found that gene-specific thresholds for these two classifiers perform well in RPGR, OCRL, RS1, COL2A1.
